app/views/layouts/application.html.slim

```diff
@@ -3,7 +3,11 @@ html
   head
     title Voluntaries Platform
     = stylesheet_link_tag 'application', media: 'all',
-    | 'data-turbolinks-track': 'reload'
+      'data-turbolinks-track': 'reload'
```

Comment:
The server will throw an error because this is invalid code, rubocop doesn't lint slim files, so there seems to be something fishy